src/client.js

```diff
@@ -149,6 +149,22 @@ export class StitchClient {
     return this.auth.provider('userpass').register(email, password, options);
   }
 
+
+  /**
+   * Links the currently logged in account with another existing account.
```

Comment:
[minor] I think it's a little misleading to say link the current account with another existing account since it can't actually link two existing users. Maybe "Links the currently logged in user with another identity", and for the `providerType` param: "the authentication provider of the identity to link with this user (e.g 'userpass', 'facebook', google')"